sklearn/neighbors/_classification.py

```diff
@@ -707,9 +714,39 @@ def predict_proba(self, X):
             The class probabilities of the input samples. Classes are ordered
             by lexicographic order.
         """
-
+        check_is_fitted(self, "_fit_method")
```

Comment:
I think this change might deserve its own PR.

Reply:
Actually we use this condition below
```python
if (
   self.weights == "uniform"
   and self._fit_method == "brute"
   and not self.outputs_2d_
   and RadiusNeighborsClassMode.is_usable_for(X, self._fit_X, metric)
):

So I think it would make sense to first call check_is_fitted. A similar approach has been used in ArgKminClassMode.
```

Reply:
Ah I was not clear enough: we have to call in it, but this should already be the case on `main` (and it isn't). :slightly_smiling_face:

Reply:
Oh right I'll make a small PR then? We would need a test as well for this?